docs/apache-airflow-providers-databricks/index.rst

```diff
@@ -80,7 +80,8 @@ PIP requirements
 PIP package                   Version required
 ============================  ===================
 ``apache-airflow``            ``>=2.1.0``
-``databricks-sql-connector``  ``>=1.0.2, <2.0.0``
+``databricks-sql-connector``  ``>=2.0.0, <3.0.0``
```

Comment:
Where can I find the source code repository for this package to compare the new and old version? Without it, it is hard to see if there is change in behaviour.

Reply:
Cc: @potiuk

Reply:
Sources are included into package itself, as dev team is still working on extraction of source code into a separate repository.  But I've put an unofficial repository with code for all released versions based on the source code from PyPi: https://github.com/alexott/databricks-sql-connector-unofficial

The new version is primarily refactoring, plus some additional connection parameters (catalog/schema/http_headers).

I've also addressed other review comments